### ISSUE TITLE
Multiple output streams from the same camera

### ISSUE THREAD

**Issue by pravardhanreddy** [AlexxIT/go2rtc#426]
First of all thank you so much for this project. Tinkering with it has helped me learn so much about ffmpeg and various media codecs.

So I have a logitech webcam which can produce mjpeg streams at 1280x960 30fps
How do I create a sub stream from the same camera with lower resolution and frame rate?

I have the following config in frigate (hw acceleration for both decoding and encoding the mjpeg to h264)
```
go2rtc:
  streams:
    hall: exec:ffmpeg -hwaccel nvdec -hwaccel_device 0 -f v4l2 -input_format mjpeg -video_size 1280x960 -framerate 30 -i /dev/video0 -pix_fmt yuv420p -c:v h264_nvenc -g 30 -profile:v high -level:v auto -preset:v p2 -tune:v ll -rtsp_transport tcp -f rtsp {{output}}
```

The stream looks really good with this config. I wanted to create a substream which I can use for detection in frigate.

I was able to get this ffmpeg command working that can create two streams simultaneously
```
ffmpeg -hwaccel nvdec -hwaccel_device 0 \
-f v4l2 -input_format mjpeg -video_size 1280x960 -framerate 30 -i /dev/video0 \
-filter_complex "[0:v]split=2[v1][v2]; \
                 [v1]format=yuv420p[v1out]; \
                 [v2]scale=640x480:flags=bilinear,fps=5[v2out]" \
-map "[v1out]" -c:v h264_nvenc -g 60 -profile:v high -level:v auto -preset:v p2 -tune:v ll \
-f rtsp -rtsp_transport tcp rtsp://10.11.0.69:8554/testcam \
-map "[v2out]" -c:v h264_nvenc -g 60 -profile:v high -level:v auto -preset:v p2 -tune:v ll \
-f rtsp -rtsp_transport tcp rtsp://10.11.0.69:8554/testcam_sub
```

I am just not sure how I can put this in the config. Any suggestions?

If there is a more efficient way of doing this, kindly let me know :)

**Comment by AlexxIT** [comment]
You can use incoming source https://github.com/AlexxIT/go2rtc#incoming-sources
But you have to manually start such a stream

**Comment by pravardhanreddy** [comment]
Sorry for the really late reply. Tested the 3 sources method and it works great. I would absolutely recommend adding this to the USB camera section of the Readme. When I started setting all this up I had a bit of a hard time getting the USB camera to work especially with MJPEG which gives higher frame rate at full resolution than RAW. I bet over 90% people with USB cams have a similar use case. Anyway, thank you so much for the tip.
